adoc/chapters/acknowledgements.adoc

```diff
@@ -96,6 +96,7 @@
   * Peter Thoman, University of Innsbruck
   * Biagio Cosenza, University of Salerno
   * Paul Preney, University of Windsor
+  * Aymeric Millan, CEA, Maison de la Simulation
```

Comment:
The organization names are alphabetized.  Can you move this up in this list after "Caster Communications" and before "Codeplay"?